Bottom: c8d8d6b4be24265eba03acc9daf111e309b61d9b
Top:    5fdb04ada7567de91b0ec969e2a8f9c8b61eb8f7
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-29 09:39:18 -0500

Refresh of fixup-asset-host

---
diff --git a/roles/buildbot-worker-ptxdist/templates/pydistutils.cfg.j2 b/roles/buildbot-worker-ptxdist/templates/pydistutils.cfg.j2
index 3c7f3345..1d205ea2 100644
--- a/roles/buildbot-worker-ptxdist/templates/pydistutils.cfg.j2
+++ b/roles/buildbot-worker-ptxdist/templates/pydistutils.cfg.j2
@@ -1,2 +1,2 @@
 [easy_install]
-index_url = {{asset_host}}
+index_url = {{asset_host}}/pypi/simple
